swagger.yaml

```diff
@@ -12,7 +12,7 @@ externalDocs:
 host: api.oregonstate.edu
```

Comment:
Can you also add `contact` field? I mainly wanna keep all APIs consistent by following the same template: https://github.com/osu-mist/express-api-skeleton/blob/master/openapi.yaml#L9-L12

Reply:
Done